data/beps_completed.yml

```diff
@@ -20,19 +22,21 @@
 
 -   number: '002'
     title: BIDS Models Specification
+    link: https://docs.google.com/document/d/1bq5eNDHTb6Nkx3WUiOBgKvLNnaa5OMcGtD0AZ9yms2M/
```

Comment:
probably better to point to URL of the current bids stats model website

this website will ultimately move when we merge all our website but let's start with valid URL for now